File: internal/content/testdata/bench/go/stock-buy-sell.go
func max_profit(prices []int) int {
	buy := prices[0]
	res := 0
	for i := 1; i < len(prices); i++ {
		profit := prices[i] - buy
		if profit > res {
			res = profit
		}
		if prices[i] < buy {
			buy = prices[i]
		}
	}
	return res
}

File: internal/content/testdata/bench/rust/stock-buy-sell.rs
fn max_profit(prices: Vec<i64>) -> i64 {
    let mut buy = prices[0];
    let mut res = 0i64;
    for i in 1..prices.len() {
        res = res.max(prices[i] - buy);
        buy = buy.min(prices[i]);
    }
    res
}
